diff --git a/node_modules/react-native-google-mobile-ads/android/src/main/java/io/invertase/googlemobileads/ReactNativeGoogleMobileAdsBannerAdViewManager.java b/node_modules/react-native-google-mobile-ads/android/src/main/java/io/invertase/googlemobileads/ReactNativeGoogleMobileAdsBannerAdViewManager.java
index 5671260..697e4b2 100644
--- a/node_modules/react-native-google-mobile-ads/android/src/main/java/io/invertase/googlemobileads/ReactNativeGoogleMobileAdsBannerAdViewManager.java
+++ b/node_modules/react-native-google-mobile-ads/android/src/main/java/io/invertase/googlemobileads/ReactNativeGoogleMobileAdsBannerAdViewManager.java
@@ -18,6 +18,7 @@ package io.invertase.googlemobileads;
  */
 
 import android.app.Activity;
+import android.graphics.Color;
 import android.view.ViewGroup;
 import androidx.annotation.NonNull;
 import com.facebook.react.bridge.Arguments;
@@ -223,6 +224,8 @@ public class ReactNativeGoogleMobileAdsBannerAdViewManager
             ? new AdManagerAdView(currentActivity)
             : new AdView(currentActivity);
 
+    adView.setBackgroundColor(Color.TRANSPARENT);
+    reactViewGroup.setBackgroundColor(Color.TRANSPARENT);
     adView.setDescendantFocusability(ViewGroup.FOCUS_BLOCK_DESCENDANTS);
     adView.setOnPaidEventListener(
         new OnPaidEventListener() {
diff --git a/node_modules/react-native-google-mobile-ads/ios/RNGoogleMobileAds/RNGoogleMobileAdsBannerView.mm b/node_modules/react-native-google-mobile-ads/ios/RNGoogleMobileAds/RNGoogleMobileAdsBannerView.mm
index 4c788ad..75ef78b 100644
--- a/node_modules/react-native-google-mobile-ads/ios/RNGoogleMobileAds/RNGoogleMobileAdsBannerView.mm
+++ b/node_modules/react-native-google-mobile-ads/ios/RNGoogleMobileAds/RNGoogleMobileAdsBannerView.mm
@@ -146,6 +146,8 @@ using namespace facebook::react;
   };
   _banner.rootViewController = [UIApplication sharedApplication].delegate.window.rootViewController;
   _banner.delegate = self;
+  _banner.opaque = NO;
+  _banner.backgroundColor = [UIColor clearColor];
 }
 
 - (void)requestAd {
@@ -158,6 +160,8 @@ using namespace facebook::react;
     return;
   } else {
     [self initBanner:GADAdSizeFromNSValue(_sizeConfig[@"sizes"][0])];
+    self.opaque = NO;
+    self.backgroundColor = [UIColor clearColor];
     [self addSubview:_banner];
     _banner.adUnitID = _unitId;
     [self setRequested:YES];
